app/controllers/patients_controller.rb

```diff
@@ -236,8 +236,10 @@ def update
 
     # Update patient history with detailed edit diff
     patient_before = patient.dup
-    Patient.detailed_history_edit(patient_before, patient, allowed_params&.keys, current_user.email) if patient.update(content)
 
+    render(json: patient.errors, status: 422) and return unless patient.update(content)
```

Comment:
I think this might cause `patient.update(content)` to be called twice

Reply:
I'm not following. Why would it be called twice?

Reply:
if `patient.update(content)` on line `240` is successful, the `patient.update(content)` on line `242` would also get called?

Reply:
Oh I see. Yeah it looks to me like we can remove the `if patient.update(content)` on line 242, but I'll wait for @ngfreiter just in case I'm missing something.

Reply:
Not missing anything, I think this snuck in after a rebase. Thank you for catching it, I will fix it.

Reply:
Waiting on this fix for approval.

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/582/commits/33d419546a9cb359df9640c01ef170160c5f5284.